src/components/AccountNumber.tsx

```diff
@@ -73,9 +73,8 @@ const styles = StyleSheet.create({
     marginBottom: 8,
   },
   link: {
-    ...fontStyles.label,
+    ...fontStyles.small,
```

Comment:
small moment of confusion: what is the reason to keep `fontStyles` here?

Reply:
oh it is because i removed the export for the font family, this component was overriding the font family here which really turns it into something similar to the `small` style. this will eventually be removed